gamebook/tests/test_options.py

```diff
@@ -9,12 +9,15 @@ def test_draw():
 
     scene1 = Scene("scene1", "welcome", [])
     scene2 = Scene("scene2", "also welcome", [])
-    nextscene1 = Decision(scene1, success_rate = 1.0)
-    nextscene2 = Decision(scene2, success_rate = 0)
-    option1 = Option(option1_title,["n"], decision = nextscene1)
-    option2 = Option("option no. 2",["m"], decision = nextscene2)
+    nextscene1 = NextScene(scene1, success_rate = 1.0)
+    nextscene2 = NextScene(scene2, success_rate = 0)
+    option1 = Option(option1_title,["n"], next_scene = nextscene1)
+    option2 = Option("option no. 2",["m"], next_scene = nextscene2)
 
     options = Options([option1, option2])
-    result = options.decision_draw()
+    result = options.next_scene_draw()
```

Comment:
You deleted options, how does this work? 
Plus I'd like to understand how it works without options, I didn't understand from your email the replacement.

Reply:
i didn't commit options because i wanted to hear your opinion first.
i added a check for when options is empty but haven't uploaded it.

Reply:
i replied to your question at gmail

Reply:
replied.